docs/fieldnorm/Software life cycle processes.md

```diff
@@ -0,0 +1,67 @@
+
+## 1.2 Software Development Life Cycle
+SDLC processes that are commonly used include planning, requirements, architectural design, development, testing, deployment, and maintenance. Laboratories should establish documented guidelines that cover all SDLC stages.
+
+## 1.2.1 Planning
+Planning entails developing a comprehensive software development plan. This includes brainstorming the initial ideas, ideation of the final goal, determining the necessary resources, creating a schedule, and establishing achievable milestones. Bioinformaticians should establish a clear communication path with stakeholders during this stage to ensure that the plan meets the goal.
+Within a laboratory, software development often occurs in the background of laboratory tests. The laboratory therefore needs to develop a solid strategy for change management for its tests that emphasizes all these features, and software development should be an integral part of this strategy.
+Smaller-scale software development and maintenance cycles that might not affect other areas of a laboratory should maintain a similar strategy on a smaller scale and be less laborious, and they might use tooling that is more effective for software development, such as code based documentation, an online documentation system that is more suitable for software development planning (like Confluence, or wiki pages), an issue management system (like issue manager or Jira) or combinations thereof.
+Guidelines should cover and define both large-scale and small-scale strategies.
+
+_Note: Examples of small scale & large scale projects could be:_\
+_-  Small-scale: the adaptation of a new component to existing software_\
+_-  Large-scale: developing entirely new software or big refactors of existing software_
+
+## 1.2.2 Requirements
+This process involves gathering, documenting, and analyzing the software requirements. It includes capturing both functional and non-functional requirements that the software must fulfill to meet the intended use and comply with regulatory standards. One example strategy for capturing this, which is also well suited for medical laboratories, is to use user stories because stakeholders within a medical laboratory have very specific needs but are often not able to translate those into well-defined technical requirements.
+
+Guidelines should support the following processes to gather, convert, track, and update the requirements during a project:
+
+  a.   Collaboration between end users and developers when defining software requirements.
+
+  b.  Convertion of users' requirements into small individual development tasks.
+
+  c.  The option to manage tasks using a task management system integrated with your Git development system.
```

Comment:
I don't get the link with requirements here

Reply:
You're right, it belongs more in the development phase. I have moved it there